tools/github-actions/get-npm-tag/src/main.ts

```diff
@@ -0,0 +1,46 @@
+import * as core from '@actions/core';
+import { getOctokit } from '@actions/github';
+import { compare, major, prerelease, valid } from 'semver';
+
+async function run(): Promise<void> {
+  try {
+    const isPreRelease = core.getInput('is-prerelease') === 'true';
+    const version = core.getInput('version', {required: true});
+    const [owner, repo] = core.getInput('repository', {required: true}).split('/');
+
+    if (!valid(version)) {
+      core.setFailed(`Invalid version (version: ${version})`);
+      return;
+    }
+
+    const releaseResponse = await getOctokit(core.getInput('token', {required: true})).rest.repos.listReleases({
+      owner,
+      repo
+    });
+    const latestVersion = releaseResponse.data
+      .filter((release) => !release.draft && !release.prerelease)
+      .map((release) => release.tag_name.replace(/^v/i, ''))
+      .sort(compare)
+      .reverse()[0];
```

Comment:
this was copied, but I'm not very fan of doing a full iteration with a `reverse()` when we could simply access the last element with `length-1` or `.pop()`